### PR TITLE
test:Signin to another server open using menu bar

### PR DESCRIPTION
**Pull request by codedsun**
#### Summary

test:Signin to another server open using menu bar

#### Ticket Link

  Fixes https://github.com/mattermost/desktop/issues/1894

#### Checklist

- [x] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
- [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
- [x] executed `npm run lint:js` for proper code formatting

#### Device Information
This PR was tested on: 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by codedsun** [comment]
@devinbinnie Please review

File: e2e/specs/menu_bar/file_menu.test.js [hunk 1]
```diff
@@ -39,4 +39,21 @@ describe('file_menu/dropdown', function desc() {
...
+
+    it('MM-T805 Sign in to Another Server Window opens using menu item', async () => {
+        if (process.platform === 'win32' || process.platform === 'linux') {
+            const mainWindow = this.app.windows().find((window) => window.url().includes('index'));
+            mainWindow.should.not.be.null;
+            robot.keyTap('alt');
+            robot.keyTap('enter');
+            robot.keyTap('f');
+            robot.keyTap('s');
+            robot.keyTap('s');
```
Comment (devinbinnie) [inline e2e/specs/menu_bar/file_menu.test.js:51]: Is the second `s` just to make sure it works?

**Review by devinbinnie — APPROVED** [review]
LGTM! Just one non-blocking comment.
Thanks @codedsun!

**Review by M-ZubairAhmed — APPROVED** [review]
Perfect looks good to me

File: e2e/specs/menu_bar/file_menu.test.js (hunk 1 above)
Comment (codedsun) [reply e2e/specs/menu_bar/file_menu.test.js:51]: No @devinbinnie The second s is to take the cursor to sign in to another server tab, first s takes it to settings.

Comment (devinbinnie) [reply e2e/specs/menu_bar/file_menu.test.js:51]: Ah, right :P

**Comment by codedsun** [comment]
Thanks team 💯

**Pull request merged**